src/plugin.ts

```diff
@@ -42,6 +42,15 @@ export abstract class ManifestPlugin {
     this.logger = logger;
   }
 
+  /**
+   * Perform post-processing on commits, e.g, sentence casing them.
+   * @param {Commit[]} commits The set of commits that will feed into release pull request.
+   * @returns {Commit[]} The modified commit objects.
+   */
+  processCommits(commits: Commit[]): Commit[] {
```

Comment:
TODO for next major version, let's run the default conventional commit parser earlier (outside of the strategy classes) and pass in the `ConventionalCommit[]` objects in. This will make it cleaner for plugin commit processors and the commit post-processing in the strategies.

Reply:
Added TODO.